pages/service-contract.md

```diff
@@ -1,14 +1,14 @@
 ---
 permalink: /service-contracts/
 layout: default
-title: Service Contracts
+title: Service contracts
 ---
 <a name="service-contract"></a>
 There is a defined support contract specifying a base-line level of service and software support.
-This should include but is not limited to:
+This should include but isn't limited to:
 
-* System configuration is managed in a version control system and should be automatically applied for consistency
-* Defined base operating system and pre-approved software repositores (like YUM or DEB) for common components like PHP or Python.
+* System configuration is managed in a version control system and should be automatically applied for consistency.
+* Defining a base operating system and pre-approved software repositores (like YUM or DEB) for common components like PHP or Python
 
-_Note that this is a work in progress; we will be adding more content
+_Note that this is a work in progress; we'll be adding more content
```

Comment:
I think we can drop this sentence now. Certainly we'll be adding more, but this is vestigial from when I created the original skeleton.